wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/romireference/RobotContainer.java

```diff
@@ -76,7 +76,20 @@ private void configureButtonBindings() {
    * @return the command to run in autonomous
    */
   public Command getAutonomousCommand() {
-    // An ExampleCommand will run in autonomous
-    return null;
+    // Change this to AutonomousTime to use a different
+    // command sequence
+    return new AutonomousDistance(m_drivetrain);
```

Comment:
Could use a SendableChooser here to switch between AutonomousTime vs AutonomousDistance